src/server/system_services/pool_server.js

```diff
@@ -48,6 +48,12 @@ const POOL_HOSTS_INFO_DEFAULTS = Object.freeze({
     by_service: {},
 });
 
+
+// key: namespace_resource_id, value: array of issues (see namespace_resource_schema)
+const map_issues_report = new Map();
+// key: namespace_resource_id, value: last_monitoring (see namespace_resource_schema)
```

Comment:
do we really need this map?
I think that having a single map that captures all the namespace_resource_monitoring info is less complex - 
// key: namespace_resource_id, value:  { last_monitoring : date, issues: array of issues }

Reply:
I removed it, hoping it is less complex now, thanks.